.gitignore

```diff
@@ -25,6 +25,7 @@ __pycache__/
 # Distribution / packaging
 .Python
 inventory/*/artifacts/
+inventory/mycluster
```

Comment:
A better approach would be to use exclusion syntax:
```
inventory/*
!inventory/local
!inventory/sample
```
That will properly exclude mycluster, artifacts, and any other user-created file.

Reply:
Changes made per your suggestion.